lib/services/geocode.js

```diff
@@ -10,10 +10,15 @@ class Geocode {
   async fetchAddress() {
     // format: latlng=40.714224,-73.961452
     const latLongString = `${this.latLong.lat},${this.latLong.long}`
-    let url = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latLongString}&key=${process.env.GOOGLE_GEOCODE_KEY}`;
-    let res = await fetch(url);
-    let googleInfo = await res.json();
-    return googleInfo
+    const url = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${latLongString}&key=${process.env.GOOGLE_GEOCODE_KEY}`
+
+    try {
+      const response = await fetch(url);
+      const json = await response.json();
+      return json
+    } catch (error) {
+      return error
+    }
```

Comment:
Added some error handling to the google geocode service because i has having a hard time stepping through the reports test to see if it was working